cpp/include/cudf/io/text/multibyte_split.hpp

```diff
@@ -30,11 +30,25 @@ namespace cudf {
 namespace io {
 namespace text {
 
+/**
+ * @brief Parsing options for multibyte_split.
+ */
+struct parse_options {
+  /**
+   * @brief Only rows starting inside this byte range will be part of the output column.
+   */
+  byte_range_info byte_range = create_byte_range_info_max();
+  /**
+   * @brief Should delimiters at the end of rows be stripped from the output column?
```

Comment:
```suggestion
   * @brief Should delimiters at the end of rows be stripped from the output column
```
The question mark seems wrong to me here. Perhaps this could be rephrased otherwise I think removing the ? is fine.

Reply:
I'd rephrase.
```suggestion
   * @brief Whether delimiters at the end of rows should be stripped from the output column
```